packages/core/src/components/dialog/_dialog.scss

```
@@ -34,6 +34,7 @@ Markup:
 Styleguide dialog
 */
 
+$pt-dialog-background-color: $light-gray4 !default;
```

this should match the naming convention of the other dialog variables:

```suggestion
$dialog-background-color: $light-gray4 !default;
```